shotover-proxy/benches/cassandra_benches.rs

```diff
@@ -1,22 +1,75 @@
-use cassandra_cpp::{stmt, Session};
+use cassandra_cpp::{stmt, Session, Statement};
 use criterion::{criterion_group, criterion_main, Criterion};
 use test_helpers::docker_compose::DockerCompose;
 
 #[path = "../tests/helpers/mod.rs"]
 mod helpers;
 use helpers::ShotoverManager;
 
+struct Query {
+    name: &'static str,
+    statement: Statement,
+}
+
 fn cassandra(c: &mut Criterion) {
     let mut group = c.benchmark_group("cassandra");
     group.throughput(criterion::Throughput::Elements(1));
     group.noise_threshold(0.2);
 
-    let statement =
-        stmt!("INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');");
+    let queries = [
+        Query {
+            name: "insert",
+            statement: stmt!(
+                "INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');"
+            ),
+        },
+        Query {
+            name: "select",
+            statement: stmt!("SELECT id, x, name FROM benchmark_keyspace.table_1;"),
+        },
+    ];
+    for query in queries {
+        // Benches the case where the message does not meet the criteria for encryption
```

Comment:
If it's not being encrypted, wouldn't it just be the same as the passthrough one?

Reply:
the point of these benches for redis cache and protect is to prove that when we dont hit the path with special logic it is as close as possible to passthrough.